src/wp-admin/includes/image.php

```diff
@@ -372,13 +407,30 @@ function wp_create_image_subsizes( $file, $attachment_id ) {
 	return _wp_make_subsizes( $new_sizes, $file, $image_meta, $attachment_id );
 }
 
+/**
+ * Gets a sources array element from a meta.
+ *
+ * @since 6.0.0
+ * @access private
+ *
+ * @param array $meta The meta to get the source from.
+ * @return array The source array element.
+ */
+function _wp_get_sources_from_meta( $meta ) {
+	return array(
+		'file'     => wp_basename( $meta['file'] ),
```

Comment:
Do we want to verify that `file` is also present in the provided array in the same way `filesize` is verified on the next line?

Reply:
added check